### PR TITLE
Add tests for tombstoned task names

### PR DESCRIPTION
**Pull request by cdonati**
This goes with https://github.com/AppScale/appscale/pull/2777.

### CODE REVIEW

**Comment by cdonati** [comment]
Good idea.

File: python27-app/module-main/taskqueue.py [hunk 1]
```diff
@@ -624,4 +657,6 @@ def post(self):
   ('/python/taskqueue/pull/lease_modification', LeaseModificationHandler),
   ('/python/taskqueue/pull/brief_lease', BriefLeaseHandler),
   ('/python/taskqueue/clean_up', CleanUpTaskEntities),
+  ('/python/taskqueue/empty_callback', EmptyCallback),
+  ('/python/taskqueue/task', TaskHandler),
```
Comment (whoarethebritons) [inline python27-app/module-main/taskqueue.py]: Not important really, but do we want these alphabetized? I can see they already weren't that's why I'm asking.

Comment (cdonati) [reply python27-app/module-main/taskqueue.py]: I think they are ordered the same as the handlers, which were just added in the order they were written. I wouldn't object if somebody wanted to alphabetize them, though.

**Comment by cdonati** [comment]
Against master: https://ocd.appscale.com:8080/job/Hawkeye/15324/
Against https://github.com/AppScale/appscale/pull/2777: https://ocd.appscale.com:8080/job/Hawkeye/15323/

**Pull request merged**